README.md

```diff
@@ -30,28 +30,18 @@ You will need the following things properly installed on your computer.
 
 * `git clone git@github.com/adfinis-sygroup/timed-frontend`
 * `cd timed-frontend`
-* `npm install`
-* `bower install`
-
-## Configurations
```

Comment:
Is the configuration part no longer needed/possible?

Reply:
Yes, it was removed in #124